tasks/packages.yml

```diff
@@ -17,8 +17,30 @@
 # limitations under the License.
 #
 
+- include: packages_ubuntu.yml
+  when: ansible_distribution == 'Ubuntu'
+
 - include: packages_redhat.yml
   when: ansible_os_family == 'RedHat'
 
-- include: packages_debian.yml
-  when: ansible_os_family == 'Debian'
+- name: Stop and disable keystone
+  service:
+    enabled: False
+    name: '{{ keystone_service }}'
+    state: stopped
+  when:
+    - not ansible_distribution == 'Ubuntu'
+    - not ansible_distribution_release == 'trusty'
+    - not ansible_virtualization_type == 'docker'
```

Comment:
I don't like adding container logic or workarounds there, is there any other way to keep idempotency ?